src/adapter/deconz/adapter/deconzAdapter.ts

```diff
@@ -724,7 +723,7 @@ class DeconzAdapter extends Adapter {
                 clusterID: resp.clusterId,
                 header,
                 data: resp.asduPayload,
-                address: resp.destAddrMode === 0x03 ? `0x${resp.srcAddr64!}` : resp.srcAddr16!,
+                address: resp.srcAddrMode === 0x03 ? `0x${resp.srcAddr64!}` : resp.srcAddr16!,
```

Comment:
Was this changed on purpose? (because this was not changed in https://github.com/Koenkk/zigbee-herdsman/commit/98e3384b74429d94a81c4d84938f133c4b9f6078#diff-e287278735156a983a1ca18677f8f6580622520aa743a7a9c191db0ed111f20d)

Reply:
Sry, forgot to mention this. Using destAddrMode for srcAddr looks like a typo. Changed this without thinking much.